main.html

```diff
@@ -189,21 +252,29 @@
 
 > 1.
 
+**2GEN:**(in this mode of solving, you can only do <R,U>, which has the moves R, R', R2, U, U', U2)
```

Comment:
Here the open and close angular brackets are not shown, because they are interpreted as HTML tags. I am fixing this.